benchmarks/models/hf_gpt2.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Run it at its parent folder, and check result at ../perf.
+# USAGE - ./benchmark.sh
+#   [fairseq|fairseq+fastseq|transformers|transformers+fastseq]
+#   <model>
+#   <task>
+#   <split> # train/val/test (text) or train/valid/test (binary)
+#   <batch-sizes>
+source hf.sh
+
+# MODEL - bart large cnn from transformer
+# TASK - cnn dm val full set
+
+./benchmark.sh \
+    transformers+fastseq \
+    gpt2 \
+    cnn_dm.1k/raw \
+    val \
+    16/32/64/128 \
+    --task summarization \
+    --no_repeat_ngram_size 3 \
+    --max_tokenizer_length 512 \
+    --max_gen_length 711
+
+./benchmark.sh \
+    transformers \
+    gpt2 \
+    cnn_dm/raw \
+    val \
+    16/32/64 \
+    --task summarization \
+    --no_repeat_ngram_size 3 \
+    --max_tokenizer_length 512 \
+    --max_gen_length 711
+
+
```

Comment:
Add rouge/bleu score and speed check

Reply:
Will add it once solving the pending issues.